src/util/runExtension.ts

```diff
@@ -338,6 +338,16 @@ Please remove the \`{{[[roam/js]]}}\` code that installed this extension and ref
             },
           },
         },
+        ui: {
+          commandPalette: {
+            addCommand: ({ label, callback, disableHotkey, defaultHotkey }) => {
+                  label
+                  callback
+                  disableHotkey
+                  defaultHotkey
+                },
```

Comment:
Is this not a compiler error?

Reply:
`npm start` throws an error, but for something else

```
[tsl] ERROR in \roamjs-components\src\types\index.ts(26,28)
      TS2307: Cannot find module 'cytoscape' or its corresponding type declarations.
ts-loader-default_199d6be0b453cdd4
```

Reply:
be sure that `@types/cytoscape` is installed locally and run `npm install` if not. should still be in the `package.json` deps